geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -756,6 +756,13 @@ When enabled, also sets ssl-endpoint-identification-enabled to true.</td>
 <td>JKS</td>
 </tr>
 
+<tr>
+<td>standard-output-always-on</td>
```

Comment:
I thought the property was `StandardOutputAlwaysOn` (pascal case) according to #5183 ? Maybe I'm not understanding how it works.

Reply:
Sorry, I missed #5197. Now it all makes sense!